bin/static-review.php

```diff
@@ -12,8 +12,8 @@
  * @see http://github.com/sjparkinson/static-review/blob/master/LICENSE
  */
 
-$included = include file_exists(__DIR__ . '/../vendor/autoload.php')
-    ? __DIR__ . '/../vendor/autoload.php'
+$included = include file_exists(__DIR__ . '/../autoload.php')
+    ? __DIR__ . '/../autoload.php'
```

Comment:
Ah, I think we are seeing a variation in the configuration of Composer's `bin-dir`. It seems very likely that we need to read the composer.json value of `bin-dir` to correctly locate the autoload file.

The current search path works for people who use `"bin-dir": "bin/"`. Your change works for people who use `vendor/bin/`.